testing/utilities.py

```diff
@@ -105,6 +105,69 @@ def compare_ref_timeseries_df(self, df, ref_filepath):
             
         return None
     
+    def compare_ref_json(self, json_test, ref_filepath):
+            '''Compare a json to a reference json saved as txt.
```

Comment:
Is the ".txt" extension used for any reason? So far we've used ".json" which I think was more clear.

Reply:
Sounds good, changed in https://github.com/ibpsa/project1-boptest/pull/189/commits/b76cfb448de10d6b64a91e7f497c6de96709156c.